package.json

```diff
@@ -27,6 +27,7 @@
     "express-sslify": "^1.2.0",
     "helmet": "^3.13.0",
     "markdown": "^0.5.0",
+    "moo": "^0.5.0",
```

Comment:
Do you have experience with this? Seems like a great  solution, but I'm always cautious about bringing in new packages that are `0.*`.

Reply:
Yeah, I'd feel better if it were 1.0 too.

I haven't used it before but am comfortable using it because it's [been around for years](https://www.npmjs.com/package/moo?activeTab=versions), [sees regular maintenance](https://github.com/no-context/moo/commits/master), and is a dependency of [nearley](https://nearley.js.org/#projects-using-nearley), a popular JavaScript  parser toolkit.

If you think further due diligence is necessary or that it's too great a risk, I'm all ears.

Reply:
No objections. Let's give it a shot!